src/plugins/mf_priority.cpp

```diff
@@ -170,6 +171,38 @@ static void add_special_association (flux_plugin_t *p, flux_t *h, int userid)
  *                                                                            *
  *****************************************************************************/
 
+/*
+ * Get config information about the various priority factor weights
+ * and assign them in the priority_weights map.
+ */
+static int conf_update_cb (flux_plugin_t *p,
+                           const char *topic,
+                           flux_plugin_arg_t *args,
+                           void *data)
+{
+    int fshare_weight = -1, queue_weight = -1;
+    flux_t *h = flux_jobtap_get_flux (p);
+
+    // unpack the various factors to be used in job priority calculation
+    if (flux_plugin_arg_unpack (args,
+                                FLUX_PLUGIN_ARG_IN,
+                                "{s?{s?{s?i, s?i}}}",
+                                "conf", "priority_factors",
```

Comment:
Suggestion: put all flux-accounting values under one heading, maybe `[accounting]`? 

Will all configurable priority factors actually be weights? If so, then maybe the sub-table should
just be `factor-weights` and the individual factor weights like `fairshare`, `queue`, `age`, etc.

So in the config file it would be
```toml
[accounting.factor-weights]
fairshare = 1
age = 1
```
and so on?

Reply:
That's a good suggestion, thanks! I've just pushed up my branch that makes this change to use a sub-table `[accounting.factor-weights]`.